doc/Uplink.xml

```diff
@@ -91,6 +91,8 @@
     <para role="reference">&lt;<link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://www.onvif.org/onvif/specs/core/ONVIF-Core-Specification.pdf"></link>&gt;</para>
     <para>ONVIF Media2 Service Specification</para>
     <para role="reference">&lt;<link xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://www.onvif.org/specs/srv/media/ONVIF-Media2-Service-Spec.pdf"></link>&gt;</para>
+    <para>ONVIF Advanced Security Service Specification</para>
```

Comment:
Didn't we choose to get rid of the Advanced adjective?

Reply:
![image](https://github.com/user-attachments/assets/b8b2e0e3-8ed7-4402-acf2-7030729e607f)

Document name still reflects "Advanced" and hence used that, I am open to change, just a clarification.

Reply:
10/24 VEWG Telco
  - Agreed to remove "Advanced" reference